torchvision/models/vision_transformer.py

```diff
@@ -148,12 +159,32 @@ def __init__(
         self.num_classes = num_classes
         self.representation_size = representation_size
         self.norm_layer = norm_layer
+        self.conv_stem_layers = conv_stem_layers
 
         input_channels = 3
```

Comment:
Looks like this could be moved inside the else statement below or hardcoded directly in line 187.